src/controllers/loginController.ts

```diff
@@ -0,0 +1,79 @@
+import express , {Request, Response} from 'express';
+import bcrypt from 'bcrypt';
+import jwt from 'jsonwebtoken';
+import passport from 'passport';
+import { Strategy as LocalStrategy } from 'passport-local';
+import dotenv from 'dotenv';
+import { testPassword } from '../utils/validations';
+import Database from '../database/db';
+import sendResponse from '../utils/response';
+
+
+//////////////////////////////////////////////////////////////////////////////////////////////////////////////////
+
+dotenv.config()
+const router = express.Router();
+router.use(passport.initialize());
+router.use(express.json())
+
```

Comment:
Please initailize this in app page because we might use it in future time

Reply:
solved